sympy/physics/control/lti.py

```diff
@@ -0,0 +1,553 @@
+        TransferFunction(p**2 - 4*p + 16, p - 4, p)
+        >>> tf2 - (b + 7)
+        TransferFunction(p - (7 - p)*(b + 7) + 1, 7 - p, p)
+
+        """
+        other = _sympify(other)
+        if other.is_number:
+            return TransferFunction(self.num - self.den*other, self.den, self.var)
+
+        elif isinstance(other, TransferFunction):
+            if not self.var == other.var:
+                raise ValueError("Both the Transfer Functions should be anchored "
+                    "with the same variable.")
+            p = self.num * other.den - other.num * self.den
+            q = self.den * other.den
+            return TransferFunction(p, q, self.var)
+
+        elif isinstance(other, Expr) and other.has(Symbol):
+            return TransferFunction(self.num - self.den*other, self.den, self.var)
+        else:
+            raise ValueError("{} cannot be subtracted from TransferFunction."
+                .format(type(other)))
+
+    def __rsub__(self, other):
+        return -self + other
+
+    def __mul__(self, other):
+        """Multiplies Transfer Functions.
+
+        Parameters
+        ==========
+
+        other : TransferFunction
+            The Transfer Function to multiply to current (self) Transfer Function.
+
+        Returns
+        =======
+
+        TransferFunction
+            The resultant Transfer Function after multiplying (with '*' operator)
+            self to other.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(s + 1, s - 8, s)
+        >>> tf2 = TransferFunction(p + s**2, p - s, s)
+        >>> tf1*tf2
+        TransferFunction((p + s**2)*(s + 1), (p - s)*(s - 8), s)
+        >>> 9*tf2
+        TransferFunction(9*p + 9*s**2, p - s, s)
+        >>> tf1*(a + 6)
+        TransferFunction((a + 6)*(s + 1), s - 8, s)
+
+        """
+        other = _sympify(other)
+        if other.is_number:
+            return TransferFunction(self.num*other, self.den, self.var)
+
+        elif isinstance(other, TransferFunction):
+            if not self.var == other.var:
+                raise ValueError("Both the Transfer Functions should be anchored "
+                    "with the same variable.")
+            p = self.num * other.num
+            q = self.den * other.den
+            return TransferFunction(p, q, self.var)
+
+        elif isinstance(other, Expr) and other.has(Symbol):
+            # other input is a polynomial.
+            if not other.is_commutative:
+                raise ValueError("Only commutative expressions can be multiplied "
+                    "with a TransferFunction.")
+            return TransferFunction(self.num*other, self.den, self.var)
+        else:
+            raise ValueError("TransferFunction cannot be multiplied with {}."
+                .format(type(other)))
+
+    __rmul__ = __mul__
+
+    def __div__(self, other):
```

Comment:
Is `__div__` required, since the support for Python 2 has been removed. I think only  `__truediv__` should be defined.

Reply:
Thanks, I just read [this](https://www.python.org/dev/peps/pep-0238/#why-is-there-a-need-for-truediv-and-itruediv).